.expeditor/config.yml

```diff
@@ -8,6 +8,12 @@ slack:
 rubygems:
   - appbundle-updater
 
+release_branches:
+  - main:
+      version_constraint: 2.*
```

Comment:
This needs to be updated to 19.* as per discussion

Reply:
done in https://github.com/chef/appbundle-updater/pull/111/commits/8463278164c913c545544c4ce82c875ebc6f8bc9